Alan_WarrenDesafio1/Program.cs

```diff
@@ -1,22 +1,31 @@
-using Alan_WarrenDesafio1.Data;
+using Alan_WarrenDesafio1.DomainServices;
 using Alan_WarrenDesafio1.Validators;
+using Application.Validators;
 using AppServices;
 using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System.Reflection;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-
 builder.Services.AddControllers()
     .AddFluentValidation(cfg =>
     {
-        cfg.RegisterValidatorsFromAssemblyContaining<CustomerValidator>();
+        cfg.RegisterValidatorsFromAssemblyContaining<CreateCustomerRequestValidator>();
+        cfg.RegisterValidatorsFromAssemblyContaining<UpdateCustomerRequestValidator>();
```

Comment:
Trocar `cfg` para `options` ou `opts`
```suggestion
        cfg.RegisterValidatorsFromAssembly(Assembly.Load("Application"));
```